docs/nextstrain-internal/pathogen-repo-ci.md

```diff
@@ -0,0 +1,54 @@
+# Pathogen Repo CI
+
+Setting up a CI GitHub Action workflow that automatically runs is extremely
+helpful for catching any changes that break pathogen workflows.
+
+The CI GH Action workflow is _not_ intended to check the validity of the results,
+it just ensures that the pathogen workflow still runs to completion.
+
+## Using `pathogen-repo-ci`
+
+Nextstrain maintains a reusable [pathogen-repo-ci][] GitHub Action workflow that
+can be used to create the CI workflow with minimum set up:
+
+```yaml
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
```

Comment:
I'd encourage adding a basic `schedule:` on here too. It's really nice to come back to a repo after a while of inactivity and know if CI has broken due to the passage of time before you do some work and expect CI to pass.

_non-blocking_

Reply:
> I'd encourage adding a basic schedule: on here too. It's really nice to come back to a repo after a while of inactivity and know if CI has broken due to the passage of time before you do some work and expect CI to pass.

True! What would be a reasonable schedule here? Once a week?

Reply:
Added in https://github.com/nextstrain/pathogen-repo-guide/pull/45/commits/b183592127529e036785db5aff1a5a2774b57414